docs/rust-version.md

```diff
@@ -4,11 +4,15 @@ Builds on CI always run against the most stable rust version, but we also suppor
 
 ## Minimum Supported Rust Version
 
-The MSRV is the version listed in the badge at the [top of the main README](https://github.com/kube-rs/kube-rs#kube-rs), and it is verified as buildable as part of the main CI action.
+The MSRV is shown in the main [Cargo.toml.rust-version key](https://github.com/kube-rs/kube-rs/blob/80272c04ae0036983bb0de4d522003cfd1f5c189/kube/Cargo.toml#L15) plus as a [**readme badge**](https://github.com/kube-rs/kube-rs#kube-rs):
```

Comment:
Is the plan to keep this link up to date over time?

Reply:
oh, hm, good point, maybe i should point it in the general direction without a hash.

Reply:
done, simpler now.